src/i18n/en-US/action.ts

```diff
@@ -70,11 +70,19 @@ const action = {
   completeAction: 'Complete action',
   completeActionWithoutEmail: 'Complete action without email',
   cancelAction: 'Cancel action and return to request',
+  error:
+    'There was a problem completing your action. Please try again. If the error persists, please try again at a later date.',
   requestEdits: {
     breadcrumb: 'Request edits',
     title: 'Action: request edits',
     description:
-      'Use this action if there are changes that need to be made to the Intake Request or Business Case form in order to proceed with the governance process. Specify the edits or additional information needed from the requester.'
+      'Use this action if there are changes that need to be made to the Intake Request or Business Case form in order to proceed with the governance process. Specify the edits or additional information needed from the requester.',
+    success: 'You have requested edits to the Draft Business Case.',
+    modal: {
+      title: 'Are you sure you want to complete this action to request edits?',
+      content:
+        '<p>If you request changes to the {{formName}}, the requester will be able to make updates to any field within that form. While they are completing those changes, they will be unable to work on any other part of the process. You will receive an email notification once the requester has resubmitted their form.</p><p>If you progress this request to a new step before they have resubmitted their form, they will lose the ability to make the requested changes.</p>'
```

Comment:
IIRC tags in the translation library arent really HTML tags.  They are placeholders for something the package calls `components` which can be more flexible when dynamically rendering context for special styling.

Reply:
I've use components in translation files for instances where only one word needs to be bolded or italicized.  I the <p> tag in the file could literally be named anything.  It's being mapped directly to a real <p> tag in the JSX <Trans /> component, so it makes sense to call it that in the file

Reply:
i18Next doesn't read that as an HTML tag - it's just a way of implementing the `Trans` component using named component keys instead of `<1 />`. I could realistically put anything within that tag. I just used `p` for simplicity because I knew it was going to be a paragraph tag.

So the `Trans` component looks like this:

```jsx
<Trans
  i18nKey="action:notification.description"
  components={{ p: <p className="line-height-body-5" /> }}
/>
```

But I could have replaced `p` with anything else as long as the component key in the `Trans` component prop matched it.